{
  "schema_version": "1.4.0",
  "id": "GHSA-2hmp-5wqg-f24h",
  "modified": "2025-03-10T15:30:47Z",
  "published": "2025-03-10T15:30:47Z",
  "aliases": [
    "CVE-2025-1497"
  ],
  "details": "A vulnerability, that could result in Remote Code Execution (RCE), has been found in PlotAI. Lack of validation of LLM-generated output allows attacker to execute arbitrary Python code.\nVendor commented out vulnerable line, further usage of the software requires uncommenting it and thus accepting the risk. The vendor does not plan to release a patch to fix this vulnerability.",
  "severity": [
    {
      "type": "CVSS_V4",
      "score": "CVSS:4.0/AV:N/AC:L/AT:N/PR:N/UI:N/VC:H/VI:H/VA:H/SC:N/SI:N/SA:N/E:X/CR:X/IR:X/AR:X/MAV:X/MAC:X/MAT:X/MPR:X/MUI:X/MVC:X/MVI:X/MVA:X/MSC:X/MSI:X/MSA:X/S:X/AU:X/R:X/V:X/RE:X/U:X"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2025-1497"
    },
    {
      "type": "WEB",
      "url": "https://github.com/mljar/plotai/commit/bdcfb13484f0b85703a4c1ddfd71cb21840e7fde"
    },
    {
      "type": "WEB",
      "url": "https://cert.pl/en/posts/2025/03/CVE-2025-1497"
    },
    {
      "type": "WEB",
      "url": "https://cert.pl/posts/2025/03/CVE-2025-1497"
    },
    {
      "type": "WEB",
      "url": "https://github.com/mljar/plotai"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-77"
    ],
    "severity": "CRITICAL",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2025-03-10T14:15:24Z"
  }
}